pages/docs/integrations/grouparoo-bigquery.mdx

```diff
@@ -1,24 +1,229 @@
+    </tr>
+    <tr>
+      <td>
+        <a href="#bigquery-import-query">
+          <strong>BigQuery Import Query</strong>
+        </a>
+      </td>
+      <td>Import or update records via a custom Bigquery query.</td>
+    </tr>
+  </tablebody>
+</table>
 
-<ValidateAndApplyConfig headingLevel="h3" />
+## Setup
 
-<PluginDocsColumnarSource plugin="BigQuery" />
+In order to use BigQuery, you’ll need to enable it as a service in your Google Cloud project. Create a service account which will be the source for a Client Email Address and Private key you will need to configure the initial connection for a BigQuery App in Grouparoo.
 
-## BigQuery Next Steps
+## App
 
-Once you have the Plugin installed, App created, and a Source configured, you are ready to [validate](/docs/cli/config#validate), [apply](/docs/cli/config#apply), then [import your data](/docs/cli/run#run)!
+To connect BigQuery to Grouparoo, you [create an App](https://www.grouparoo.com/docs/config/apps) to tell Grouparoo how to connect to Postgres. This specifies the connection details for the database you want to sync data to or from.
+
+### App Options
+
+In Grouparoo, provide the options that best describe how you would like to connect to BigQuery.
+
+<ImageInBrowserFrame
+  src="/images/docs/integrations/bigquery/app/options.png"
+  alt="BigQuery App Options"
+  width={733}
+  height={499}
+/>
+<br />
+
+<table className="docTable">
+  <tablebody>
+    <tr>
+      <td>App Option</td>
+      <td>Description</td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Project ID </strong>(required)
+      </td>
+      <td>Project id from Google.</td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Dataset</strong> (required)
+      </td>
+      <td>Default dataset id to use for sources.</td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Client Email Address</strong> (required)
+      </td>
+      <td>
+        Email of service account. This will follow the pattern of the helper
+        text pre-populated in the field and is generated by Google when you
+        create a service account for this project.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Private Key </strong>(required)
+      </td>
+      <td>Private key of service account.</td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Location</strong>
+      </td>
+      <td>
+        Region or multi-region code of your dataset. ("US", "EU", "asia-east1",
+        etc.) This field is required for datasets outside the US.
+      </td>
+    </tr>
+  </tablebody>
+</table>
+
+## BigQuery Import Table
+
+This source reads from a single schema or table in your BigQuery database using a primary or foreign key. Properties are generated from this table from row values or aggregations of row values.
+
+### Options
+
+<ImageInBrowserFrame
+  src="/images/docs/integrations/bigquery/import-table/options.png"
+  alt="BigQuery Import Table Options"
+  width={733}
+  height={100}
+/>
+<br />
+
+### Mapping
+
```

Comment:
could have a sentence that links to the core documentation.

Reply:
Good idea!